spec/requests/feedback_pages_spec.rb

```diff
@@ -313,7 +313,7 @@
 
         it "redirects to homepage" do
           current_path.should == root_path
-          page.should have_selector('.flash-alert', text:"You are not authorized to access this page.")
+          page.should have_selector('.flash-alert', text: "You are not authorized to access this page.")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.